### PR TITLE
Bug fix for FAQ

### PR DESCRIPTION
**Pull request by Jordan231111**
# Fixed accordion not collapsing after opening.

## Rewrote the entire logic of the component
- Replaced the input type="radio" element with input type="checkbox". This allows each FAQ item to be independently toggled.
- Added a checked prop to the <input> element, which is set to expandedIndex === index. This checks if the current index matches the expandedIndex state.
- Added an onChange event handler to the <input> element. When the checkbox is toggled, it calls the handleToggle function with the current index.
- Implemented the handleToggle function, which updates the expandedIndex state. If the clicked index is the same as the currently expanded index, it sets expandedIndex to null (closing the FAQ item). Otherwise, it sets expandedIndex to the clicked index (expanding the clicked FAQ item).

### CODE REVIEW

**Comment by CooperW824** [comment]
I think he was asking for the ability to open multiple FAQs at the same time. The state management you added doesn't allow for that. Although the fix is pretty easy if you remove the states you added, you can open multiple FAQs at the same time.

**Comment by CooperW824** [comment]
Also, it looks like your commits removed the comments from the files? Was this intentional?

File: components/faq/faq.tsx [hunk 1]
```diff
@@ -61,41 +61,45 @@ const faqs: FAQ[] = [
 
 const FAQPage = () => {
 	const [highlightFAQ, setHighlightFAQ] = useState(false);
+	const [expandedIndex, setExpandedIndex] = useState<number | null>(null);
```
Comment (CooperW824) [inline components/faq/faq.tsx:64]: This state management prevents the user from being able to open multiple FAQs at the same time. Which is how understood his feedback.

File: components/faq/faq.tsx [hunk 2]
```diff
@@ -61,41 +61,45 @@ const faqs: FAQ[] = [
...
 		let faqEnd = faqStart + (document.getElementById("faq")?.offsetHeight || window.innerHeight);
 
-		// Update whether the faq should be highlighted when the user scrolls
 		const handleScroll = () => {
 			setHighlightFAQ(window.scrollY > faqStart && window.scrollY < faqEnd);
 			faqStart = (document.getElementById("faq")?.offsetTop || window.innerHeight) - 140;
 			faqEnd = faqStart + (document.getElementById("faq")?.offsetHeight || window.innerHeight);
 		};
 
-		// Update the faqStart and faqEnd when the user resizes the window
```
Comment (CooperW824) [inline components/faq/faq.tsx:77]: Your commits removed the comments. Was this intentional?

**Review by CooperW824 — CHANGES_REQUESTED** [review]

**Comment by Jordan231111** [comment]
> I think he was asking for the ability to open multiple FAQs at the same time. The state management you added doesn't allow for that. Although the fix is pretty easy if you remove the states you added, you can open multiple FAQs at the same time.

No he mentioned could not close it and I was able to see that the faq could not be closed

**Comment by CooperW824** [comment]
Ok, that's fine, was it intentional to remove the comments from the files?

**Review by CooperW824 — APPROVED** [review]

**Pull request merged**